drivers/hmis_external_apis/app/models/hmis_external_apis/ac_hmis/exporters/client_export.rb

```diff
@@ -0,0 +1,95 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HmisExternalApis::AcHmis::Exporters
+  class ClientExport
+    attr_accessor :output
+
+    def initialize(output = StringIO.new)
+      require 'csv'
+      self.output = output
+    end
+
+    def run!
+      Rails.logger.info 'Generating content of client export'
+
+      write_row(columns)
+      total = clients.count
+
+      Rails.logger.error "There are #{total} clients to export. That doesn't look right" if total < 10
+
+      clients.find_each.with_index do |client, i|
+        Rails.logger.info "Processed #{i} of #{total}" if (i % 1000).zero?
+
+        client_values = client_columns.map do |col|
+          client.send(col)
+        end
+
+        mci =
+          client.external_ids.to_a.find do |x|
+            x.namespace == HmisExternalApis::AcHmis::Mci::SYSTEM_ID
+          end
+        external_id_values = [mci&.value]
```

Comment:
```suggestion
        # if the client has multiple MCI IDs, it doesn't matter which one we send
        external_id_values = [client.ac_hmis_mci_ids&.first]
```

Reply:
Good idea. I've pushed something that captures that (had to make a few tiny changes)